app/enterprise/2.1.x/vitals/vitals-prometheus-strategy.md

```diff
@@ -53,9 +53,15 @@ using default config that listens on port `9090`.
 
 ### Download and configure StatsD exporter
 
-The latest release of StatsD exporter can be found at the
-[Bintray](https://bintray.com/kong/statsd-exporter). The binary is distributed
-with some specific features like min/max gauges and Unix domain socket support.
+The latest release of StatsD exporter can be pulled from Docker Hub with the
+following command: 
+
+```sh
+docker pull kong/statsd-exporter-advanced:0.3.1
+```
+
+The binary is distributed with some specific features like min/max gauges
+and Unix domain socket support.
```

Comment:
```suggestion
The binary includes features like min/max gauges and Unix domain 
socket support.
```
I know you didn't write or edit this line, but adjusting it because I added the suggestion to use "distributed" above. Won't get into trying to figure out what "some specific features" really means, that'll go too deep down a rabbit hole.